src/Illuminate/Database/Query/Processors/SqlServerProcessor.php

```diff
@@ -9,20 +9,16 @@
 class SqlServerProcessor extends Processor
 {
     /**
-     * Process an "insert get ID" query.
+     * Process a "last insert ID" query.
      *
      * @param  \Illuminate\Database\Query\Builder  $query
-     * @param  string  $sql
-     * @param  array   $values
      * @param  string  $sequence
      * @return int
      */
-    public function processInsertGetId(Builder $query, $sql, $values, $sequence = null)
+    public function processLastInsertId(Builder $query, $sequence = null)
     {
         $connection = $query->getConnection();
 
-        $connection->insert($sql, $values);
```

Comment:
Look at the single commit "Add method Builder::lastInsertId()".
Then `processInsertGetId` is identical to the parent one, and I remove it in the following commit "Same as parent".